.devcontainer/local/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "nextflow-training (local)",
+
+    "build": {
+        "dockerfile": "../Dockerfile",
+        "context": "..",
+        "options": ["--platform=linux/amd64"]
```

Comment:
Do we really want to force people to use amd64? Is this needed?

Reply:
It actually was. I got some weird Docker shim error without it.

Reply:
But maybe if you build the arm image it won't be needed

Reply:
https://github.com/nextflow-io/training/pull/549